packages/webdriverio/src/commands/element/shadow$$.js

```diff
@@ -0,0 +1,22 @@
+import {shadowFnFactory} from '../../scripts/shadowFnFactory'
+/**
+ *
+ * Access elements inside a given element's shadowRoot
+ *
+ * <example>
+    :shadow$$.js
+    it('should return elements inside a shadowRoot', () => {
+        const innerEl = $('.input').shadow$$('#innerEl');
+        console.log(innerEl.getValue()); // outputs: 'test123'
+    });
+ * </example>
+ *
+ * @alias element.shadow$$
+ * @uses commands/$$
+ * @type action
+ *
+ */
+
+export default async function shadowRoot (selector) {
+    return await this.$$(shadowFnFactory(selector, true))
```

Comment:
same

Reply:
it's imported above the comment block. Easy enough to move it if needed